VideoWeb/VideoWeb.UnitTests/AuthenticationSchemes/QuickLinksSchemeTests.cs

```diff
@@ -0,0 +1,100 @@
+using FluentAssertions;
+using Microsoft.AspNetCore.Authentication.JwtBearer;
+using NUnit.Framework;
+using System;
+using System.IdentityModel.Tokens.Jwt;
+using VideoWeb.AuthenticationSchemes;
+using VideoWeb.Common.Configuration;
+
+namespace VideoWeb.UnitTests.AuthenticationSchemes
+{
+    public class QuickLinksSchemeTest
+    {
+        private QuickLinksScheme _sut;
+        private QuickLinksConfiguration _configuration;
+
+        [SetUp]
+        public void SetUp()
+        {
+            _configuration = new QuickLinksConfiguration
+            {
+                Issuer = "issuer",
+                JwtProviderSecret = "x4p5Kxsygx3dYAso0JKZljK0PL926mxppc5gGqeV9aRydc++gSNx4UITuZ1G6YJX7KgymQnQiEsaG/XIUKTPPA=="
```

Comment:
Hope this is not used in any environments secrets.

Reply:
It doesn't appear to be relevant to the tests so I have changed to a generic value.